src/amoc_metrics.erl

```diff
@@ -1,66 +1,96 @@
 -module(amoc_metrics).
 
+-export([default_reporting_opts/0]).
 -export([new_histogram/1, new_counter/1, new_spiral/1]).
 -export([new_histogram/2, new_counter/2, new_spiral/2]).
 -export([update_hist/2, update_counter/2, update_spiral/2]).
+-export([messages_spiral_name/0, message_ttd_histogram_name/0]).
 
--type name() :: atom().
+-type name() :: atom() | [atom()].
```

Comment:
Do we need both options here? It seems error-prone.

Reply:
Yes we do, because of the existing dashboards on Tide.